astrology.lic

```diff
@@ -2,19 +2,25 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#astrology
 =end
 
-custom_require.call(%w(common common-arcana drinfomon spellmonitor))
+custom_require.call(%w(common common-arcana common-travel drinfomon spellmonitor))
 
 class Astrology
   include DRC
   include DRCA
+  include DRCT
 
   def initialize
     settings = get_settings
     @constellations = get_data('constellations').constellations
     @have_telescope = settings.have_telescope
+    @ap_training = settings['astral_plane']['train_in_ap']
+    @ap_source = settings['astral_plane']['train_source']
```

Comment:
It looks like some of the sub-settings of `astral_plane` are used in astrology.lic and others in bescort.lic, which could be confusing.